VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionTopic.java

```diff
@@ -80,4 +81,20 @@ public boolean containsThread(@NonNull DiscussionThread discussionThread) {
         }
         return false;
     }
+
+    @NonNull
+    public List<String> getAllTopicIds() {
+        final List<String> ids = new ArrayList<>();
+        appendTopicIds(ids);
+        return ids;
+    }
+
+    private void appendTopicIds(List<String> ids) {
```

Comment:
If this method is not used anywhere else, then it's logic could be merged inside `getAllTopicIds()`?